docs/code-examples/repository-pattern/01_simple_example/ilGeoLocationRepository.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+class ilGeoLocationRepository {

```

Comment:
Please write an interface that is named like this, rename the implementation here, e.g. to `ilGeoLocationDBRepository`, and let it `implement` the new interface. IMO the interface is the real thing and not the implementation, so the interface deserves to get the name.

Reply:
Changed it. I created a new folder called 00_base_classes and added `ilGeoLocationRepository` as interface and renamed the original class.